### DISCUSSION TITLE
Contributions not getting Register in GitHub Profile graph.

### DISCUSSION THREAD

**Discussion by goswamianshuman** [community/community#69410] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Is this issue due to any sort of maintenance work behind the scenes?🤔 if not then please have a look over this problem unless it occurs with other developers in the community.🙂

(GitHub graph referred in above conversation)
![image](https://github.com/community/community/assets/68741416/e274ab8c-5057-43bb-9479-e7f8f38c5fbf)

**Comment by himanshu-03** [comment, marked as answer]
Hello @goswamianshuman !!

I faced this issue recently in my account as well. However, after observing this I noticed that GitHub takes some time to show the contributions we make on any org's repo and sometimes personal repos too.

Please wait for a day or two and you will see all your contributions updated on the specific days the contributions were made.
